Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @summary Helper functions to display errors in the format MSBuild can understand.
+ */
+
+const os = require('os');
+
+function handleErrorObjectInner(error, type, defaultCode) {
+    if (error === null || error === undefined) {
+        return handleErrorObjectInner(Error("Missing error argument"), 'error', 'META-ERROR');
+    }
+
+    const code = error.code || defaultCode;
+    const path = error.path || 'no-path';
+    const message = (error.message || error)?.toString().replace(/^Error[ :]+/i, '');
+    const line = 'line' in error ? error.line : 1;
+    const column = 'column' in error ? error.column : 1;
```

Comment:
Any reason why we can't use `||` here, too?
```suggestion
    const line = error.line || 1;
    const column = error.column || 1;
```

Reply:
If the number would be 0 for any reason, your change would turn it into 1. Also if the value is `null`, `false`, etc I'd rather output that to aid with debugging by showing the invalid input.

Reply:
Fair enough.